src/components/Todo.js

```diff
@@ -0,0 +1,40 @@
+function Todo(vname) {
+  let name = vname;
+  let description = null;
+  let date = null;
+  let priority = 'default';
+  let checked = false;
+  // const timeStamp = ;
+  let id = `${vname.toLowerCase().replace(/\W/g, '')}${new Date().getTime()}`;
+  this.setId = (name) => {
+    id = `${name.toLowerCase().replace(/\W/g, '')}${new Date().getTime()}`;
```

Comment:
- [ ] Currently, your project doesn't work in development and instead throws an error. It might be an issue with these lines. 
![ice_screenshot_20201021-113221](https://user-images.githubusercontent.com/33323077/96709289-66748f80-1392-11eb-9c5b-4a87bbc280ce.png)